Backends/CLX/mirror.lisp

```diff
@@ -1,11 +1,29 @@
 (in-package :clim-clx)
 
+(defun %set-window-name (window name)
+  (setf (xlib:wm-name window) name)
+  (xlib:change-property window
+                        :_NET_WM_NAME
+                        (babel:string-to-octets name :encoding :utf-8)
+                        :UTF8_STRING 8))
+
+(defun %set-window-icon-name (window icon-name)
+  (setf (xlib:wm-icon-name window) icon-name)
+  (xlib:change-property window
+                        :_NET_WM_ICON_NAME
+                        (babel:string-to-octets icon-name :encoding :utf-8)
+                        :UTF8_STRING 8))
+
+(defmethod port-set-mirror-name ((port clx-basic-port) mirror name)
+  (%set-window-name mirror name)
+  (%set-window-icon-name mirror name))
+
```

Comment:
do we want to introduce sheet protocol for names?

i.e something in this spirit:

```
(defclass named-sheet-mixin ()
  ((sheet-name :initarg :name :accessor sheet-name)))

(defmethod (setf sheet-name) :after ((sheet mirrored-sheet) new-name)
  (setf (port-set-mirror-name (sheet-direct-mirror sheet))))

(defmethod (setf sheet-name) :after ((sheet named-sheet-mixin) new-name)
  (note-sheet-name-changed sheet new-name))
```

Reply:
I'm all for it. My main difficulty with this would be knowing where to put (which file, which package, etc.) the pieces.

Reply:
all three forms would belong in sheets.lisp after sheet-mirrored-mixin and before sheets ads bounding rectangle.